Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -265,14 +242,10 @@ AppDelegateViewModelOutputs {
       .filter { AppEnvironment.current.currentUser != nil }
 
     if #available(iOS 10.0, *) {
-      self.getNotificationAuthorizationStatus =
-        Signal.merge(
-          applicationLaunchedWithUser,
-          self.notificationAuthorizationCompletedProperty.signal)
+      self.getNotificationAuthorizationStatus = applicationLaunchedWithUser
 
-      self.registerForRemoteNotifications = self.notificationAuthorizationStatusProperty.signal
-        .skipNil()
-        .filter { $0.isAuthorized }
+      self.registerForRemoteNotifications = self.notificationAuthorizationCompletedProperty.signal
+        .filter(isTrue)
         .ignoreValues()
```

Comment:
thinking about this some more... It seems like `registerForRemoteNotifications` should also be called as you have it @bormind  (for the use case that the user originally denies notifications, but then allows them in their device Settings). It was unclear to me, but it seems like a device can be registered, but they won't see the alert again after the first time.